elibrary-core/source/src/view/panels/com/javafee/elibrary/core/settingsform/SystemParametersPanel.java

```diff
@@ -120,9 +120,9 @@ public SystemParametersPanel() {
 		add(lblApplicationGeneratedPasswordLength, gbc_lblApplicationGeneratedPasswordLength);
 
 		spinnerApplicationGeneratedPasswordLength = new CustomJSpinner(
-				Constants.SPINNER_INITIAL_VALUE,
-				Constants.SPINNER_MINIMUM_VALUE,
-				Constants.SPINNER_MAXIMUM_VALUE);
+				Constants.APPLICATION_MIN_PASSWORD_LENGTH,
```

Comment:
Instead of using 'password', 'generated password' should be used in Constants, because that change is related to the generated password, not existing ones or these provided during f.eg. registration or login.

Reply:
My fault, at that case generated password min length is equal to APPLICATION_MIN_PASSWORD_LENGTH.